### ISSUE TITLE
Bar chart horizontal orientation out of X axis

### ISSUE THREAD

**Issue by Always-prog** [apache/superset#23453]
### Vertical orientation is ok
![image](https://user-images.githubusercontent.com/66589759/226851521-99c6bfc0-b2c0-4583-bcb2-f55921272047.png)

### Horizontal orientation is weird 
![image](https://user-images.githubusercontent.com/66589759/226851738-8438b5a8-4eb4-4dd0-952b-85f3fa46b6b2.png)

## Description
I using latest version of apache Superset (latest commit on March 22).

**Comment by zhicwu** [comment]
Same issue exists in 3.0.0.

**Comment by prasanjeetdas** [comment]
Guys, any update on the above problem?

**Comment by rusackas** [comment]
Yeah, that's an odd one. It's still an issue in 4.0, but there seems to be strangely little outcry about this. PRs welcome, of course!

**Comment by rusackas** [comment]
It looks like this is resolved in master... or at least I'm unable to reproduce it now. Closing as fixed, but happy to reopen if anyone finds it still being an issue!
